.travis.yml

```diff
@@ -1,5 +1,6 @@
 language: java
 sudo: false
+dist: trusty
```

Comment:
This change was for addressing: https://travis-ci.community/t/error-installing-oraclejdk8-expected-feature-release-number-in-range-of-9-to-14-but-got-8/3766